components/webfield/MessageReviewersModal.js

```diff
@@ -171,15 +186,19 @@ const MessageReviewersModal = ({
       {error && <div className="alert alert-danger">{error}</div>}
       {currentStep === 1 ? (
         <>
-          <p>{`You may customize the message that will be sent to the ${prettyField(
-            reviewerName
-          ).toLowerCase()}. In the email
+          <p>
+            {messageOption?.value === 'allAuthors'
+              ? `You may customize the message that will be sent to authors. You can also use {{fullname}} to personalize the recipient full name. If your message is not specific to ${submissionName.toLowerCase()}, please email from the author group.`
```

Comment:
Just a small change:
```suggestion
              ? `You may customize the message that will be sent to authors. You can also use {{fullname}} to personalize the recipient full name. If your message is not specific to a ${submissionName.toLowerCase()}, please email from the author group.`
```